src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -864,6 +883,17 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
       file_github_issue = info.get('file_github_issue', False)
       job.environment_string += f'FILE_GITHUB_ISSUE = {file_github_issue}\n'
 
+      # Device-specific queues created during project setup.
+      queue_id = info.get('queue_id', False)
+      if queue_id:
+        platform = info.get('platform',
+                            'ANDROID').upper() + ':' + queue_id.upper()
```

Comment:
No, this is not necessary. This is simply adding `ANDROID` as the default platform if one was not specified.

Alternatively, I could change this to a different platform `None` / `LINUX` or raise an error if platform was not specified. Since Android is the only project currently using it I thought it made sense to implement this way, but definitely willing to modify if you think one of the other 2 routes is more appropriate.

Reply:
Can you please change the default to 'LINUX'? That's most likely the common platform on ClusterFuzz, and for Android I'd expect you'd always be setting `platform`.

Reply:
Done.